xbmc/interfaces/json-rpc/VideoLibrary.cpp

```diff
@@ -1124,7 +1124,26 @@ void CVideoLibrary::UpdateVideoTag(const CVariant &parameterObject, CVideoInfoTa
   if (ParameterNotNull(parameterObject, "track"))
     details.m_iTrack = (int)parameterObject["track"].asInteger();
   if (ParameterNotNull(parameterObject, "rating"))
-    details.m_fRating = parameterObject["rating"].asFloat();
+  {
```

Comment:
Was there a reason for this change?
This change means one can no longer update just votes, but needs to update rating _and_ votes, so breaks existing tools for no apparent reason. I'd prefer to avoid this and just revert the grouping.
@phate89?